webui/react/src/e2e/models/hew/Nameplate.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseComponent, NamedComponent } from 'e2e/models/BaseComponent';
+
+/**
+ * Returns a representation of the Nameplate component from Hew.
+ * This constructor represents the contents in hew/src/kit/Nameplate.tsx.
+ * @param {object} obj
+ * @param {implementsGetLocator} obj.parent - The parent used to locate this Nameplate
+ * @param {string} obj.selector - Used instead of `defaultSelector`
+ */
+export class Nameplate extends NamedComponent {
+  override defaultSelector: string = '[class^="Nameplate_base"]'; // must be provided
+
+  #nameSelector = '.ant-typography:last-of-type';
```

Comment:
`last-of-type` seems like a fragile way to define a selector. Do we have a way forward to use another selector? (potentially in a future change if it's involved)

Reply:
our way forward would be with a change to hew. i felt clever coming up with this solution though!